BLUESPAWN-client/src/hunt/hunts/HuntT1060.cpp

```diff
@@ -63,12 +63,36 @@ namespace Hunts {
 		int detections = 0;
 		
 		for(auto& key : RunKeys){
-			for(auto& detection : CheckKeyValues(HKEY_LOCAL_MACHINE, key)){
+			for (auto& detection : CheckKeyValues(HKEY_LOCAL_MACHINE, key)) {
 				if (EvaluateFile(detection.ToString(), reaction)) {
```

Comment:
Don't use ToString, use std::get<std::wstring>(detection.value.data)

Reply:
gotcha, did not know this was the best way to do it